modules/ROOT/partials/module-loading/bundling-rollup-es6-npm_editor.adoc

```diff
@@ -9,13 +9,14 @@ Example `+src/editor.js+`
 /* Import TinyMCE */
 import tinymce from 'tinymce';
 
-/* Default icons are required. Also import custom icons if applicable */
+/* Default icons are required. After that, import custom icons if applicable */
 import 'tinymce/icons/default';
 
-/* A theme is also required */
+/* Required TinyMCE components */
 import 'tinymce/themes/silver';
+import 'tinymce/models/dom';
 
-/* Import the skin */
+/* Import a skin (can be your own instead of the default) */
```

Comment:
Should we try and avoid saying 'your own...' in the comment here and in the other similar files as well?
Possible alternative?
```suggestion
/* Import a skin (can be a custom skin instead of the default) */
```

Reply:
ah yeah we are avoiding that language aren't we. I'll do a pass over it.